estuary_updater/handlers/errata.py

```diff
@@ -0,0 +1,86 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals
+
+from estuary_updater.handlers.base import BaseHandler
+from estuary.models.errata import Advisory
+from estuary.models.bugzilla import BugzillaBug
+
+import requests
+import requests_kerberos
+import iso8601
+
+
+class ErrataHandler(BaseHandler):
+    """A handler for dist-git related messages."""
+
+    @staticmethod
+    def can_handle(msg):
+        """
+        Determine if this handler can handle this message.
+
+        :param dict msg: a message to be analyzed
+        :return: a bool based on if the handler can handle this kind of message
+        :rtype: bool
+        """
+        return msg['topic'] == '/topic/VirtualTopic.eng.errata.activity.status'
+
+    def handle(self, msg):
+        """
+        Handle an Errata tool message and update Neo4j if necessary.
+
+        :param dict msg: a message to be processed
+        """
+        advisory_id = msg['body']['headers']['errata_id']
+
+        url = 'https://errata.devel.redhat.com/api/v1/erratum/{}'.format(advisory_id)
```

Comment:
Make this a setting in the fedmsg configuration instead so that we can customize in different environments. For instance, add the base URL of the API `https://errata.devel.redhat.com/api/v1/`.